antarest/eventbus/business/redis_eventbus.py

```diff
@@ -0,0 +1,57 @@
+import json
+import logging
+import dataclasses
+import redis
+
+from typing import Any, List, Optional
+from dataclasses import dataclass
+from redis.client import Redis
+
+from antarest.common.interfaces.eventbus import Event
+from antarest.eventbus.business.interfaces import IEventBusBackend
+
+logger = logging.getLogger(__name__)
+REDIS_STORE_KEY = "events"
+
+
+@dataclass
+class RedisConfig:
+    host: str
+    port: int = 6379
+
+
+def parse_config(redis_config: Any) -> RedisConfig:
+    return RedisConfig(**redis_config)
+
+
+class RedisEventBus(IEventBusBackend):
```

Comment:
What is the purpose of redis ? Anyone seems to subscribe to redis, redis pubsub is just used to save events before launch in batch.